lib/GADS/Schema/Result/File.pm

```diff
@@ -183,6 +219,23 @@ sub export_hash
 # Created by DBIx::Class::Schema::Loader v0.07043 @ 2015-11-13 16:02:57
 # DO NOT MODIFY THIS OR ANYTHING ABOVE! md5sum:266q8eJmPiCjcNhwddaUkw
 
+sub _build_valuefield { undef; }
+
+# I've done this to override the role - I don't know how else to best implement this!
```

Comment:
I think better to use Moo's `after`, especially as the `value` will still need nulling anyway?